lib/ansible/plugins/lookup/sequence.py

```diff
@@ -180,17 +186,18 @@ def sanity_check(self):
             if self.count != 0:
                 self.end = self.start + self.count * self.stride - 1
             else:
-                self.start = 0
-                self.end = 0
-                self.stride = 0
-            del self.count
+                return False
         if self.stride > 0 and self.end < self.start:
             raise AnsibleError("to count backwards make stride negative")
         if self.stride < 0 and self.end > self.start:
             raise AnsibleError("to count forward don't make stride negative")
+        if self.stride == 0:
+            return False
```

Comment:
Out of all newly added lines, this one is not covered by any tests according to https://app.codecov.io/gh/ansible/ansible/commit/e14f9fe725e1fb1cf37a0aac932d9b9c1f1c65a3#451adf36166d1bf6560a11cf5e0ccf76-R195. If you can, please send a follow-up PR with tests hitting it so we know it works.

Reply:
Sure. Thanks